src/main/resources/org/jenkinsci/plugins/displayurlapi/DefaultProviderGlobalConfiguration/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:section title="${%Default notification URL handler}">
```

Comment:
Not sure about naming. The per-user configuration is "Notification URL", but that seems a bit inaccurate since in (almost) all cases the URLs are always `.../display/redirect`, we only change what handles that URL in Jenkins.

Reply:
I decided to use the `Default notification URL` to match the user preference, see [37a40bd](https://github.com/jenkinsci/display-url-api-plugin/pull/202/commits/37a40bd7a97e4986ff3e9ec6a974dea831262eb5).